tests/arch/arm/arm_irq_advanced_features/src/arm_dynamic_direct_interrupts.c

```diff
@@ -35,6 +35,8 @@ void test_arm_dynamic_direct_interrupts(void)
 	 * in the ROM ISR table.
 	 */
 	ARM_IRQ_DIRECT_DYNAMIC_CONNECT(DIRECT_ISR_OFFSET, 0, 0, no_reschedule);
+	/* Disable the interrupt */
```

Comment:
Maybe rephrase to, "Ensure the IRQ is disabled before enabling it at run time"

Reply:
done